packages/node/src/concordium/block.concordium.ts

```diff
@@ -0,0 +1,114 @@
+// Copyright 2020-2023 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: GPL-3.0
+
+import { getLogger } from '@subql/node-core';
+import {
+  ConcordiumBlock,
+  ConcordiumTransactionFilter,
+  ConcordiumTransactionEventFilter,
+  ConcordiumBlockFilter,
+  ConcordiumBlockWrapper,
+  ConcordiumTransaction,
+  ConcordiumTransactionEvent,
+  ConcordiumSpecialEvent,
+  ConcordiumSpecialEventFilter,
+} from '@subql/types-concordium';
+
+const logger = getLogger('block');
+
+export class ConcordiumBlockWrapped implements ConcordiumBlockWrapper {
```

Comment:
I'd like to drop this class, it seems to add an extra layer that I don't think is needed. I've recently dropped it. with ethereum